packages/contracts/test/unit/conditionalTransfer.spec.ts

```diff
@@ -64,21 +64,21 @@ contract("ConditionalTransfer", (accounts: string[]) => {
       const randomTarget = Utils.randomETHAddress();
       const tx = ct.interface.functions.executeSimpleConditionalTransfer.encode(
         [
-          makeCondition(Utils.zeroBytes32, true),
+          makeCondition(Utils.ZERO_BYTES32, true),
           {
             amount: [Utils.UNIT_ETH],
             assetType: 0,
             to: [randomTarget],
-            token: Utils.zeroAddress,
-            data: Utils.zeroAddress
+            token: Utils.ZERO_ADDRESS,
+            data: Utils.ZERO_ADDRESS
           }
         ]
       );
 
       await delegateProxy.functions.delegate(
         ct.address,
         tx,
-        Utils.highGasLimit
+        Utils.HIGH_GAS_LIMIT
```

Comment:
Any way to avoid using this so often?

Reply:
The new version of Ethers uses `estimateGas` so we can remove HIGH_GAS_LIMIT in most places. I've found we still need it for special cases though e.g. deploying via registry.